docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2'
+
+services:
+  api:
+    build:
+      context: server
+      dockerfile: Dockerfile
+    ports:
+      - 9292:9292
+    environment:
+      - RACK_ENV=production
+      - MONGODB_URI=mongodb://mongodb:27017/kontena_development
+      - VAULT_KEY=asdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdf
+      - VAULT_IV=kljhkljhkljhkljhkljhkljhkljhkljhkljhkljhkljhkljhkljhkljhkljhkljh
+      - ACME_ENDPOINT=https://acme-staging.api.letsencrypt.org/
+    volumes:
+      - ./server:/app
```

Comment:
Are these volume mappings really needed? Build should not take too long if Gemfile.lock does not change.